src/workloads/execution/UnshardedLookup.yml

```diff
@@ -1,31 +1,56 @@
 SchemaVersion: 2018-07-01
 Owner: "@mongodb/query"
 Description: |
-  Benchmark a $lookup from one unsharded collection to another in a sharded environment.
+  Benchmark a $lookup in a sharded environment where the foreign collection unsharded.
 
 Actors:
+- Name: CreateShardedCollections
+  Type: AdminCommand
+  Threads: 1
+  Phases:
+  - Repeat: 1
+    Database: admin
+    Operations:
+    - OperationMetricsName: EnableSharding
+      OperationName: AdminCommand
+      OperationCommand:
+        enableSharding: &db test
+    - OperationMetricsName: ShardSourceCollection
+      OperationName: AdminCommand
+      OperationCommand:
+        shardCollection: test.Collection2
+        key: {a: hashed}
+        numInitialChunks: 8
```

Comment:
It was kind of arbitrary. I picked it so that there wouldn't be too many or too few documents per chunk. In the case of two shards though we will get an even distribution.

Reply:
Sorry, one final comment here. Would you mind adding an operation here to disable the balancer? Had another brief conversation about it in my original PR. Wouldn't hurt to have it here too!